GOVERNANCE.md

```diff
@@ -54,6 +54,14 @@ If a vote is taken during a WG meeting, the follow rules will be followed:
   [here](https://docs.google.com/spreadsheets/d/1bw5s9sC2ggYyAiGJHEk7xm-q2KG6jyrfBy69ifkdmt0/edit#gid=0).
   Members must acknowledge their presence verbally, meaning, adding yourself
   to the "Attendees" section of the Agenda document is not sufficient.
+* When a vote is called, unless all viting members have voted, the vote will be
+  tallied no less than one week after calling the vote.
+* Voting process:
+  * Comment on the PR: "agree, yes, +1", "disagree, no, -1", or "abstain".
```

Comment:
Let's make this a bit more concrete and less chance of someone using the word "agree" w/o realizing they're voting.
`Comment on the PR: LGTM, NOT LGTM or ABSTAIN`